pkg/camo/proxy.go

```diff
@@ -21,6 +21,39 @@ import (
 	"github.com/cactus/go-camo/pkg/htrie"
 
 	"github.com/cactus/mlog"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// Subsystem for proxy metrics in Prometheus.
+const proxySubsystem = "proxy"
+
+var (
+	contentLengthExceeded = promauto.NewCounter(
```

Comment:
This creates global counters that all proxy instances would share, right?
Truth be told, not sure anyone out there is using more than one instance (the only consumer is probably cmd/go-camo). :thinking:

Reply:
Yes, it creates a global counter at the process level. That's a good question, do we want to have separate metrics if a single process is creating multiple `Proxy` instances.

One requirement would be that if we want separate metrics for each instance, they would need a label to identify them. We would need to add this to the config.

Reply:
I'll think about it a bit more. Multiple proxy instances is a _probably_ a bit of "not going to need it" at this point, because it seems unlikely there is a need for multiple instances -- not impossible, but probably unlikely.

Reply:
We can always refactor the metrics later if it turns out we need a label to identify multiple proxies per process.